samples/go/nomsfs/README.md

```diff
@@ -100,3 +110,17 @@ struct Commit {
 ...
 ```
 
+## Limitations
+
+Hard links are not supported at this time, but may be added in the future.
+Mounting a dataset in multiple locations is not supported, but may be added in the future.
+
+## Troubleshooting
+
+`Mount failed: no FUSE devices found`
+Make sure FUSE is installed. If you're on macOS make sure the kernel module is loaded by executing `/Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs`.
```

Comment:
I tried uninstalling and reinstalling multiple times, but it never fixed the problems. Once I did this load_osxfusefs everything worked. In talking with @rafael-atticlabs this was also what fixed his problems.